src/content/docs/zh-cn/recipes.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: 更多方案
+description: 为Astro添加特性的示例。
+
+---
+
+import RecipesNav from '~/components/RecipesNav.astro';
+
+为你的Astro项目中添加特性的示例。
```

Comment:
This sentence seems to be grammatically incorrect. It should be modified to:
```
为你的 Astro 项目添加特性的示例。
```
or
```
在你的 Astro 项目中添加特性的示例。
```